libraries/botframework-streaming/tsconfig.json

```diff
@@ -1,7 +1,7 @@
 {
   "compilerOptions": {
     "target": "es6",
-    "lib": ["es2015"],
+    "lib": ["es2015", "dom"],

```

Comment:
We actually took this out because it was forcing botkit to build with the `"dom"` lib which was causing issues. Is this a must-have?

Reply:
https://github.com/howdyai/botkit/issues/1894

Reply:
Fixed, good catch.